app/wizards/schools/ects/change_email_address_wizard/check_answers_step.rb

```diff
@@ -0,0 +1,29 @@
+module Schools
+  module ECTs
+    module ChangeEmailAddressWizard
+      class CheckAnswersStep < Step
+        def previous_step = :edit
+        def next_step = :confirmation
+
+        def current_email = ect_at_school_period.email
+        def new_email = store.email
+
+        def save!
+          ApplicationRecord.transaction do
+            ect_at_school_period.update!(email: new_email)
+            Events::Record.record_teacher_email_changed_event!(
```

Comment:
The timeline wording looks a bit off. It says "unchanged" even though I changed it 🤔

<img width="675" height="487" alt="image" src="https://github.com/user-attachments/assets/203d4b43-ff0a-4680-a51f-b794ba524101" />

Reply:
This happened because the old email was actually the new email (we record the event after updating).

I've added a unit test and ensured we pass the correct email 👍🏼